### PR TITLE
Pull request for WAZO-1094-switchboard-optimize-answer

### PR DESCRIPTION
**Pull request by sduthil**
switchboards: test wazo-confd cache invalidation
switchboards: implement cache for getting switchboards in wazo-confd
switchboard tests: randomize resource ids
integration tests: silence docker debug

### CODE REVIEW

File: wazo_calld/plugins/switchboards/plugin.py [hunk 1]
```diff
@@ -23,10 +27,17 @@ def load(self, dependencies):
...
         ari = dependencies['ari']
         bus_publisher = dependencies['bus_publisher']
+        bus_consumer = dependencies['bus_consumer']
         config = dependencies['config']
         token_changed_subscribe = dependencies['token_changed_subscribe']
 
         confd_client = ConfdClient(**config['confd'])
+        switchboard_get_cache = ConfdClientGetUUIDCacheDecorator(confd_client.switchboards.get, 'switchboard')
+        confd_client.switchboards.get = switchboard_get_cache
+        line_get_cache = ConfdClientGetUUIDCacheDecorator(confd_client.lines.get, 'lines')
```
Comment (fblackburn1) [inline wazo_calld/plugins/switchboards/plugin.py]: Line only have ID, not UUID

Comment (sduthil) [reply wazo_calld/plugins/switchboards/plugin.py]: True, but the type of ID does not impact the `ConfdClientGetUUIDCacheDecorator`

Comment (sduthil) [reply wazo_calld/plugins/switchboards/plugin.py]: You're right, the cache invalidation won't work with the message body with key `id`.

Comment (sduthil) [reply wazo_calld/plugins/switchboards/plugin.py]: It's fixed.

**Comment by sduthil** [comment]
Zuul is not active on this repo yet. You want to merge it?

**Comment by fblackburn1** [comment]
> Zuul is not active on this repo yet. You want to merge it?

arg ... Yeah i will pass the tests on my computer and merge after

**Pull request merged**